sholes/roster.json

```diff
@@ -35,6 +35,10 @@
         "github": "anaxz",
         "materials": [{
             "type": "cv"
+          },
+          {
+            "type": "video",
+            "url": "https://www.youtube.com/watch?v=drvQAtbSZvw"
           }]    
```

Comment:
Can you please replace 
            "url": "https://www.youtube.com/watch?v=drvQAtbSZvw"
with      "url": "drvQAtbSZvw"